service/src/directory.js

```diff
@@ -43,7 +46,7 @@ exports.getUsers = async (req) => {
       }
       page = { size: 100, token: nextToken };
     }
-    return users;
+    return create(GetObjectsResponseSchema, { results: users})
```

Comment:
I think this is meant to wrap the combined array results with a connectrpc wrapper, so that it can be serialized to json using `toJson` by the caller.  Is there a way to hide this in some way?

Perhaps we should have an accessor on the response which returns the results as a plain old json object?